src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -333,6 +343,34 @@ private RouteFW wrapRoute(
         return routeRO.wrap(buffer, index, index + length);
     }
 
+    // TODO: REMOVE CAPABILITIES FROM BEGIN
+    //       Change HttpSignalExFW -> HttpChallengeExFW
+    private long resolveChallengeDelta(
+            JsonWebSignature verified,
+            long expiresAtMillis)
+    {
+        long challengeDelta = 0;
+
+        try
+        {
+            if (verified != null)
+            {
+                final JwtClaims claims = JwtClaims.parse(verified.getPayload());
+                final NumericDate challengeAfterDate = claims.getNumericDateClaimValue(
+                        config.geClaimNamespace() + config.getClaimNameChallengeResponseTimeout());
```

Comment:
Note that `challengeResponseTimeout` is relative, so no longer need to compute delta relative to `expiresAtMillis`.

Also, the unit is now `seconds`, so it will need to be converted to `millis`.